src/components/wallet/create/ConfirmPassphrase.tsx

```diff
@@ -77,9 +58,42 @@ const ConfirmPassphrase = ({ goToNextStep, formik }: Props) => {
         evt: React.ChangeEvent<HTMLInputElement>,
         index: number,
     ) => {
-        formik.setFieldValue(`confirmPassphrase[${index}]`, evt.target.value);
+        const confirmPassphrase = values.confirmPassphrase;
+        confirmPassphrase[index] = evt.target.value;
+
+        formik.setFieldValue(`confirmPassphrase`, confirmPassphrase);
+
+        browser.runtime.sendMessage({
+            type: 'SET_LAST_SCREEN',
+            screenName: ScreenName.CreateWallet,
+            data: {
+                step: 1,
+                mnemonic: values.passphrase.join(' '),
+                network: values.wallet?.networkId(),
+                coin: values.wallet?.network().coin(),
+                confirmationNumbers: values.confirmationNumbers,
```

Comment:
these should change when the user returns to the passphrase screen (they did prior to this PR) so they are randomly generated when you navigate back and forth (to avoid people not writing it down and simply going back and forth between the two screens). Currently it doesn't do this anymore

Reply:
this comment is more for the `step 0` persisting screen down below

Reply:
@goga-m it depends on the screen, so in high level:
- step 0 (passphrase generation) -> persist passphrase, and generate new confirmation numbers
- step 1 (confirmation) -> persist existing confirmation numbers and filled in words, so if you reopen the extension these remain the same
- if you navigate back to step 0 after being at step 1, it should forget the prior confirmation numbers (and anything that was filled in) and regenerate them as you were on the passphrase step (0)